x-pack/test/api_integration/apis/transform/delete_transforms.ts

```diff
@@ -0,0 +1,263 @@
+        const { body } = await supertest
+          .post(`/api/transform/delete_transforms`)
+          .auth(
+            USER.TRANSFORM_POWERUSER,
+            transform.securityCommon.getPasswordForUser(USER.TRANSFORM_POWERUSER)
+          )
+          .set(COMMON_REQUEST_HEADERS)
+          .send({
+            transformsInfo,
+          })
+          .expect(200);
+
+        expect(body[transformId].transformJobDeleted.success).to.eql(true);
+        expect(body[transformId].destIndexDeleted.success).to.eql(false);
+        expect(body[transformId].destIndexPatternDeleted.success).to.eql(false);
+        await transform.api.waitForIndicesToExist(destinationIndex);
+        await transform.api.waitForTransformJobNotToExist(transformId);
+      });
+
+      it('should return 403 for unauthorized user', async () => {
+        const transformsInfo: TransformEndpointRequest[] = [{ id: transformId }];
+        await supertest
+          .post(`/api/transform/delete_transforms`)
+          .auth(
+            USER.TRANSFORM_VIEWER,
+            transform.securityCommon.getPasswordForUser(USER.TRANSFORM_VIEWER)
+          )
+          .set(COMMON_REQUEST_HEADERS)
+          .send({
+            transformsInfo,
+          })
+          .expect(403);
+        await transform.api.waitForIndicesToExist(destinationIndex);
+        await transform.api.waitForTransformJobToExist(transformId);
+      });
+    });
+
+    describe('single transform deletion with invalid transformId', function () {
+      it('should return 404 with error message if invalid transformId', async () => {
+        const transformsInfo: TransformEndpointRequest[] = [{ id: 'invalid_transform_id' }];
+        const { body } = await supertest
+          .post(`/api/transform/delete_transforms`)
+          .auth(
+            USER.TRANSFORM_POWERUSER,
+            transform.securityCommon.getPasswordForUser(USER.TRANSFORM_POWERUSER)
+          )
+          .set(COMMON_REQUEST_HEADERS)
+          .send({
+            transformsInfo,
+          })
+          .expect(200);
+        expect(body.invalid_transform_id.transformJobDeleted.success).to.eql(false);
+      });
+    });
+
+    describe('bulk deletion', function () {
+      const transformsInfo: TransformEndpointRequest[] = [
+        { id: 'bulk_delete_test_1' },
+        { id: 'bulk_delete_test_2' },
+      ];
+
+      beforeEach(async () => {
+        await createTransformJob('bulk_delete_test_1', 'bulk_delete_test_1');
+        await transform.api.createIndices('bulk_delete_test_1');
+        await createTransformJob('bulk_delete_test_2', 'bulk_delete_test_2');
+        await transform.api.createIndices('bulk_delete_test_2');
+      });
+
+      afterEach(async () => {
+        await transform.api.deleteIndices('bulk_delete_test_1');
+        await transform.api.deleteIndices('bulk_delete_test_2');
+      });
+
+      it('should delete multiple transform jobs by transformIds', async () => {
```

Comment:
Remove `jobs`

Reply:
Fixed in https://github.com/elastic/kibana/pull/67922/commits/d3c37b5c0d03c2ca6e211a3ff2380154223c2f0d